core/bloom_test.go

```diff
@@ -103,8 +103,6 @@ func TestBloomFilter(t *testing.T) {
 }
 
 func TestGetOrCreateBloomFilter(t *testing.T) {
```

Comment:
Not sure why these changes are showing here. These were merged already 🤔

Reply:
You may need to pull the master into your branch.

Reply:
fixed now. I had to hit Sync Fork on my fork.